generic/tests/cfg/kdump.cfg

```diff
@@ -84,8 +85,8 @@
             kdump_rsa_path = '/root/.ssh/kdump_id_rsa.pub'
             auth_key_path = '/root/.ssh/authorized_keys'
             authorized_key_cmd = "/bin/echo -e "%s" >> ${auth_key_path}"
-            host_pwd = redhat
-            guest_pwd = redhat
+            host_pwd = ${password}
```

Comment:
This is more like an example at upstream, `password` is only be defined for guest.

Reply:
So is it better not to make changes?

Reply:
Yes, you can keep them, or keep host_pwd, because host password for each user should not the same, but guest_pwd can be ${password}

Reply:
Thanks a lot, corrected.